modules/ROOT/partials/docker/spelling-service/spelling-service-installation.adoc

```diff
@@ -0,0 +1,258 @@
+[[installation]]
+== Installation
+
+[IMPORTANT]
+A valid access token is **required** in order to retrieve On-Premises services images from {companyname} Cloud Docker Registry.
+link:https://www.tiny.cloud/contact/[Contact {companyname} Support] to request the access token.
+
+=== Retrieve Docker Image
+
+. Log into the {companyname} Cloud Docker Registry: 
++
+[source, sh, subs="attributes+"]
+----
+docker login -u tiny -p [access-token] registry.containers.tiny.cloud
+----
+
+. Pull the Docker Image from the Docker registry:
++
+[source, sh, subs="attributes+"]
+----
+docker pull registry.containers.tiny.cloud/spelling-tiny:<VERSION>
+----
+
++
+[NOTE]
+Currently, the Docker images are only supported on x86-64 (also known as AMD64) architecture processors.
+
++
+Replace `<VERSION>` with `latest` or the specific version number.
```

Comment:
Suggest switching this `admon` to be mentioned after the replace with statement.

```suggestion
+
Replace `<VERSION>` with `latest` or the specific version number.
+
[NOTE]
Currently, the Docker images are only supported on x86-64 (also known as AMD64) architecture processors.
```